executor/runtime/docker/docker_linux.go

```diff
@@ -97,6 +97,12 @@ var systemServices = []serviceOpts{
 		required:     true,
 		enabledCheck: shouldStartServiceMesh,
 	},
+	{
+		humanName:    "abmetrix",
+		unitName:     "titus-abmetrix",
+		required:     true,
```

Comment:
Can you make this `false`, so that if the systemd unit doesn't work for *any* reason, we never fail a task?
Esp in these early days. I don't think any failed launch of abmetric should fail a task.

Reply:
done

Reply:
i changed this back to true to match what was going on in sshd. thoughts @solarkennedy @sargun ?

Reply:
I still say that this is a nice to have, and should never be in the critical path of a task launching.
I don't want a bad push to this container to cause tasks to fail, so therefore I think it should be `required: false`.

(I dare say sshd should be the same! But not on this PR)

Reply:
i'm not entirely sure what setting required=false means here.  But I did it =)

Reply:
When `required: false` is here, when the executor does `systemctl start titus-abmetrix`, and it doesn't work, the task continues to launch. That way, if we make a mistake in the image or the binary fails to start for whatever reason, customers are not impacted.